src/styles/main.scss

```diff
@@ -0,0 +1,65 @@
+@import "./extends";
+@import "./variables";
+
+body {
+  margin: 0;
+  background-color: #2d5074;
+}
+
+.stopwatch {
+  height: 100vh;
+}
+
+.clock {
+  width: $clock-width;
+  height: $clock-height;
```

Comment:
If clock is always a circle, you can use one variable for width and height, same for the clock center

Reply:
alternatively you can just skip setting the size and make your content dictate the clock's size